src/include/network/services/routing.rb

```diff
@@ -424,43 +437,65 @@ def validateRouting(_key, _event)
       false
     end
 
+    # Mangles route definition for storing in routes file
```

Comment:
"Mangling" is not something you want to introduce. See http://www.thefreedictionary.com/mangle (C++ mangling is, well, C++, not a good role model)
"Adapt" or "convert".